projects/tech-zone/visual-studio-code-guide.md

```diff
@@ -88,6 +88,7 @@ Once the clone is complete, you get to the main GitHub Desktop screen.
 
     ![Visual Studio Code Navigation](/media/visual-studio-code-guide_vsc-navigation.png)
 
+1. As you edit, you can see an in-app preview of the page by typing the following key combination: Ctrl+K, then type V. 
```

Comment:
Ctrl+K for Windows, for Mac OS combination is Command+V. I would also suggest to always escape command using `command`